Source/Core/Core/Debugger/BranchWatch.h

```diff
@@ -110,7 +113,7 @@ enum class BranchWatchPhase : bool
   Reduction,
 };
 
-class BranchWatch final  // Class is final to enforce the safety of GetOffsetOfRecordingActive().
+class BranchWatch
```

Comment:
I would prefer keeping it final and just removing the comment. If there's nobody wanting to inherit from it, there's no reason to have it be non-final.